src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -134,6 +134,28 @@ describe('Test suite for dictionary concepts components', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should not call the createNewName function if new names are already available', () => {
```

Comment:
@dkayiwa The bug was there because `createNewName()` was being called when a new name was already available. I have therefore **added a test** to ensure that when a new name is present, the function is not called. i.e. **not**.toHaveBeenCalled()